recipes/basis_set_exchange/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "basis_set_exchange" %}
+{% set version = "0.8.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 78e2cf3d9b6862e82b27e019c09812e01d40b785014a3969fd51b52079c90122
+
+build:
+  noarch: python
+  number: 0
+  script: "python -m pip install . -vv"
```

Comment:
use canonical form in `example/meta.yaml`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35

`script: "{{ PYTHON }} -m pip install . -vv"`

Also it looks like this project has python entry points that should be listed:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and then tested under the `tests` section:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
I added the entry points to the build section and added `cmd -h` to the testing block, which is my reading of the docs